lib/paper_trail/frameworks/active_record.rb

```diff
@@ -2,3 +2,7 @@
 # since otherwise the model(s) will get loaded in via the `Rails::Engine`.
 require "paper_trail/frameworks/active_record/models/paper_trail/version_association"
 require "paper_trail/frameworks/active_record/models/paper_trail/version"
+
+ActiveSupport.on_load(:active_record) do
```

Comment:
I think this needs to remain within `lib/paper_trail.rb` because this file only conditionally gets loaded when `ActiveRecord` is utilized outside of the context of rails.  Or perhaps there should be an `lib/paper_trail/frameworks/active_model.rb`. Regardless not sure it belongs as part of this PR, seems unrelated. Happy to make a PR for that piece separately if you'd like.